$NetBSD$

etags is installed as emacs-etags, so that it does not collide with the etags
in the base system.  etc/ctags.1 is a .so link to the etags manual page and has
to follow the rename, or man ctags finds nothing.

--- etc/ctags.1.orig	2008-10-10 10:35:49.000000000 +0900
+++ etc/ctags.1
@@ -1,3 +1,3 @@
-.so man1/etags.1
+.so man1/emacs-etags.1
 
 .\" arch-tag: 54d4579b-9d66-4ba5-9fda-f01ec83612ad
